test/Scripts.Integration.Test/SentrySetup.cs

```diff
@@ -0,0 +1,89 @@
+using UnityEditor;
+using UnityEditor.PackageManager.Requests;
+using UnityEditor.PackageManager;
+using UnityEngine;
+using System.Linq;
+
+[InitializeOnLoad]
+public static class Startup
+{
+    static AddRequest AddRequest;
+    static ListRequest ListRequest;
+
+    const string SentryPackageName = "io.sentry.unity";
+    const string SentryUPMUrl = "https://github.com/getsentry/unity.git";
+
+    static void LogDebug(string message)
+        => Debug.LogFormat(LogType.Log, LogOption.NoStacktrace, null, $"Sentry setup: {message}");
+
+    static void LogError(string message)
+        => Debug.LogFormat(LogType.Error, LogOption.NoStacktrace, null, $"Sentry setup: {message}");
+
+    static void ExitIfBatchMode(int code)
+    {
+        if (Application.isBatchMode)
+        {
+            EditorApplication.Exit(code);
+        }
+    }
+    static Startup()
+    {
+        LogDebug("checking if Sentry is installed");
+        CheckIfSentryIsInstalled();
+    }
+
+    static void RequestSentryInstall()
+    {
+        AddRequest = Client.Add(SentryUPMUrl);
+        EditorApplication.update += SentrySetupProgress;
+    }
+
+    static void CheckIfSentryIsInstalled()
+    {
+        ListRequest = Client.List();
+        EditorApplication.update += SentryIsInstalledProgress;
+    }
+
+    static void SentrySetupProgress()
+    {
+        if (AddRequest.IsCompleted)
+        {
+            EditorApplication.update -= SentrySetupProgress;
+
+            if (AddRequest.Status == StatusCode.Success)
+            {
+                LogDebug("SUCCESS");
```

Comment:
What about something more descriptive? So we don't accidently match some other test cases "success" while adding more integration/smoke tests

```suggestion
                LogDebug("Package Installation: SUCCESS");
```